src/runtime/vdom/test/scoped-slot.spec.tsx

```diff
@@ -755,4 +755,26 @@ describe('scoped slot', () => {
     // expect(root.firstElementChild.firstElementChild.children[1].firstElementChild.children[1].nodeName).toBe('GOAT');
     // expect(root.firstElementChild.firstElementChild.children[1].firstElementChild.children[1].textContent).toBe('hey goat!');
   });
+
+  it('should hide the slot\'s fallback content when slot content passed in', async() => {
+    @Component({ tag: 'fallback-test', scoped: true })
```

Comment:
So this happens when setting `scoped: true`? I think it would also happen when adding `shadow: false`. Could you also add a test for that?